CODEOWNERS

```diff
@@ -1,4 +1,33 @@
-/core/ @se3000 @RyanRHall @spooktheducks @samsondav @j16r @connorwstein @PiotrTrzpil @archseer
+# Services
+/core/services/bulletprooftxmanager @PiotrTrzpil @samsondav
+/core/services/cron @spooktheducks
+/core/services/directrequest @PiotrTrzpil @spooktheducks @j16r @connorwstein
+/core/services/eth @PiotrTrzpil @spooktheducks
+/core/services/fluxmonitorv2 @jkongie @PiotrTrzpil @spooktheducks @connorwstein
+/core/services/gasupdater @samsondav @PiotrTrzpil
+/core/services/headtracker @PiotrTrzpil
+/core/services/health @archseer
+/core/services/job @spooktheducks @connorwstein
+/core/services/keeper @RyanRHall
+/core/services/log @PiotrTrzpil @spooktheducks
+/core/services/offchainreporting @spooktheducks @connorwstein
+/core/services/periodicbackup @PiotrTrzpil
+/core/services/pipeline @spooktheducks @connorwstein
```

Comment:
Can you list me under the pipeline as well?

Reply:
Done